pyproject.toml

```diff
@@ -10,6 +10,7 @@ python = "^3.8"
 pymarc = "^4.1.1"
 apache-airflow = "^2.1.4"
 apache-airflow-providers-amazon = "^2.2.0"
+SQLAlchemy = "^1.3.18"
```

Comment:
This pinning was required to fix the `table not found` errors that are seen in the build. This fixed it locally, but does not appear to be installing the correct version in circle. Perhaps cached somehow?